tests/rustdoc-ui/doctest/check-attr-test.stderr

```diff
@@ -1,4 +1,4 @@
-error: unknown attribute `compile-fail`. Did you mean `compile_fail`?
+error: unknown attribute `compile-fail`. There is an attribute with a similar name: `compile_fail`?
```

Comment:
Why is the suggestion in the error message? I think it should be in a help suggestion (maybe even a machine-application suggestion).

Reply:
Also, it's not a question anymore.
```suggestion
error: unknown attribute `compile-fail`. There is an attribute with a similar name: `compile_fail`
```

Reply:
Good point, thanks!

Reply:
And done! :)